packages/wasm-dpp/src/state_transition/errors/invalid_state_transition_error.rs

```diff
@@ -0,0 +1,32 @@
+use dpp::consensus::ConsensusError;
+use wasm_bindgen::{prelude::wasm_bindgen, JsValue};
+
+use crate::errors::consensus_error::from_consensus_error_ref;
+
+#[wasm_bindgen(js_name = InvalidStateTransitionError)]
+pub struct InvalidStateTransitionError {
```

Comment:
I recommend to rename this struct to `InvalidStateTransitionErrorWasm` for clarity, as with all other errors

Reply:
Ok